duckduckgo/macos-browser
# Address autofill security concerns with copy changes

Task/Issue URL: https://app.asana.com/0/1199230911884351/1207411921782782/f

**Description**:
[✓ Implement Survey for Password Manager Users](https://app.asana.com/0/72649045549333/1206568003117818) showed that a proportion of users are hesitant to use our Password Manager because they don't know how secure it is. Easing these concerns should increase the adoption of DuckDuckGo's Password Manager.
These changes update and add copy to more clearly explain the security safe-guards of using the Password Manager.

**Steps to test this PR**:
- Go to the screens from the designs in [Figma](https://www.figma.com/design/wAWx1a0mAooj6sDCmoTFbS/Password-Manager-security?node-id=192-10952&node-type=FRAME&t=0sLE1hdNaQCkC7A8-0) and check they match. Double check Ship Review for any copy divergences.

**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
